### DISCUSSION TITLE
Email not sending, but no error in logs

### DISCUSSION THREAD

**Discussion by Johnson90512** [firefly-iii/firefly-iii#6716] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [ ] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [ ] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I can't get email working no matter what I do. I have verified port forwarding, 
<img width="851" alt="Screenshot 2022-12-11 212606" src="https://user-images.githubusercontent.com/25690600/206947776-5b1bf84d-0b15-4f4a-95af-952fb52000f7.png">

### Expected behaviour

I should be able to send an email and receive it to me gmail account

### Debug information

I'm not sure how to get to this page

**Comment by Johnson90512** [comment]
Also, I removed the password, but it is in the configuration

**Comment by JC5** [comment]
As per the documentation in the .env file, mail_mailer should be "smtp".

**Reply by Johnson90512** [reply]
This is definitely a step in the right direction, but still not sending hmm

**Reply by taker218** [reply]
The smtp port via ssl should be 465. In your screenshot it's 467

If you use TLS it is 587.
